language/stdlib/modules/AccountLimits.move

```diff
@@ -354,6 +354,19 @@ module AccountLimits {
         aborts_if spec_window_reset(receiving).window_inflow + amount > max_u64();
```

Comment:
Is it possible to rewrite the code so that overflow cannot happen?  I managed to do it for DesignatedDealer.  It may be helpful to look over there for coding patterns.

Reply:
Also, we are trying to always have an abort code with aborts_if specification.

Reply:
I'm actually not sure whether this is a good idea, because it has the tendency to hide programming errors. If things are out of bound, instead of a specific error, you may get some strange other error. 

In general, if this is going to happen, because it is a significant code change (I know because I've written those aborts here a while ago), I would not mix it into this PR. Perhaps in DesignatedDealer it was a moderate change, but here it will be quite a thing.